app/src/main/AndroidManifest.xml

```diff
@@ -112,6 +112,8 @@
         <activity android:name="org.mifos.mobile.ui.activities.PrivacyPolicyActivity"
                   android:windowSoftInputMode="adjustResize"/>
 
+        <activity android:name=".ui.activities.MainActivity"/>
```

Comment:
we can use `HomeActivity` if it clashes with the old one. `NewHomeActivity`

Reply:
The `MainActivity` is the one containing the `BottomNavigation` which contains the four fragments `Home`, `Account`, `Transfer` and `Profile`. Renaming it to `HomeActivity` will signify only the `HomeFragment`. Please let me know your take on this and I'll do accordingly.